<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>465</height>
   </rect>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <property name="windowTitle">
   <string>QWC2 Themes Manager</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-57</y>
         <width>438</width>
         <height>454</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="themes_tab">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <attribute name="title">
            <string>Themes</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="2" column="0" colspan="2">
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="1" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Default Scales</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="defaultScales_lineEdit">
                <property name="text">
                 <string>1000000,500000,250000,100000,50000,25000,10000,5000,2500,1000,500</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="defaultPrintScales_lineEdit"/>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="defaultPrintResolutions_lineEdit"/>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Default Theme</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="defaultTheme_comboBox"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Default Print Scales</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Default Print Resolutions</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="0" rowspan="2" colspan="2">
             <widget class="QFrame" name="frame">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="3" column="1">
                <widget class="QPushButton" name="openProject_button">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="toolTip">
                  <string>Open the themes project in QGIS.</string>
                 </property>
                 <property name="text">
                  <string>Open Theme Project</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QPushButton" name="deleteTheme_button">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="toolTip">
                  <string>Remove the selected theme.</string>
                 </property>
                 <property name="text">
                  <string>Remove Theme</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="addTheme_button">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="toolTip">
                  <string>Add the current QGIS project as a QWC2 theme.</string>
                 </property>
                 <property name="text">
                  <string>Publish current Project</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="editTheme_button">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="toolTip">
                  <string>Edit the selected theme properties.</string>
                 </property>
                 <property name="text">
                  <string>Edit Theme</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QPushButton" name="showQWC2_button">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="toolTip">
                  <string>Show the selected theme in QWC2.</string>
                 </property>
                 <property name="text">
                  <string>Show Theme in QWC2</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" rowspan="6">
                <widget class="QListWidget" name="themes_listWidget">
                 <property name="sortingEnabled">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QDialogButtonBox" name="buttonBox">
              <property name="standardButtons">
               <set>QDialogButtonBox::Retry|QDialogButtonBox::Save</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="settings_tab">
           <attribute name="title">
            <string>Settings</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Path to the QWC2 installation directory (required)</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLineEdit" name="projectsDir_lineEdit"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Path to the QGIS Server publication directory (required)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="qwc2Dir_lineEdit"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>QWC2 URL (optional)</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="1">
             <widget class="QToolButton" name="qwc2Dir_button">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QToolButton" name="projectsDir_button">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLineEdit" name="qwc2Url_lineEdit">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
